ai/core/UnityCatalog_Functions_Quickstart.ipynb

```diff
@@ -0,0 +1,737 @@
+    "unitycatalog-server-1  | # | |  | |_ __  _| |_ _   _  | |     __ _| |_ __ _| | ___   __ _  #\n",
+    "unitycatalog-server-1  | # | |  | | '_ \\| | __| | | | | |    / _` | __/ _` | |/ _ \\ / _` | #\n",
+    "unitycatalog-server-1  | # | |__| | | | | | |_| |_| | | |___| (_| | || (_| | | (_) | (_| | #\n",
+    "unitycatalog-server-1  | #  \\____/|_| |_|_|\\__|\\__, |  \\_____\\__,_|\\__\\__,_|_|\\___/ \\__, | #\n",
+    "unitycatalog-server-1  | #                      __/ |                                __/ | #\n",
+    "unitycatalog-server-1  | #                     |___/               v0.3.0-SNAPSHOT  |___/  #\n",
+    "unitycatalog-server-1  | ###################################################################\n",
+    "unitycatalog-server-1  |\n",
+    "```\n",
+    "\n",
+    "At this point, you can interface with the server as shown in this tutorial."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "755f060a-2717-4919-9e95-f661f5157b87",
+   "metadata": {},
+   "source": [
+    "## Configuration\n",
+    "\n",
+    "This next cell sets up access to your running UnityCatalog server, creates a Catalog and a Schema that we will be using throughout the remainder of this tutorial. \n",
+    "\n",
+    "> Note: The unitycatalog-client is an aiohttp-based package. When directly interfacing with the APIs in that package, make sure to use (as shown below with the `create_catalog` function) async interfaces when making calls. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "2a29fc72-8346-431b-904b-82910523b72b",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "SchemaInfo(name='AISchema', catalog_name='AICatalog', comment='This is a schema used for storing GenAI functions.', properties={}, full_name='AICatalog.AISchema', owner=None, created_at=1732297125510, created_by=None, updated_at=1732297125510, updated_by=None, schema_id='4925f53c-216a-44e7-9185-34c39fb9f51f')"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from unitycatalog.ai.core.oss import UnitycatalogFunctionClient\n",
+    "from unitycatalog.client import ApiClient, CatalogsApi, Configuration, SchemasApi\n",
+    "from unitycatalog.client.exceptions import NotFoundException\n",
+    "from unitycatalog.client.models import CreateCatalog, CreateSchema\n",
+    "\n",
+    "CATALOG = \"AICatalog\"\n",
+    "SCHEMA = \"AISchema\"\n",
+    "\n",
+    "config = Configuration()\n",
+    "config.host = \"http://localhost:8080/api/2.1/unity-catalog\"\n",
+    "client = ApiClient(configuration=config)\n",
+    "\n",
+    "catalogs_api = CatalogsApi(api_client=client)\n",
+    "schemas_api = SchemasApi(api_client=client)\n",
+    "uc_client = UnitycatalogFunctionClient(uc=client)\n",
+    "\n",
+    "\n",
+    "async def create_catalog(catalog_name, catalog_api, comment=None):\n",
```

Comment:
This part seems too complex for users to get started, can we provide them as util functions instead? Or can we create a UCClient that can handle catalog/schemas 🤔 Can be a follow-up though!

Reply:
I DO like the idea of adding synchronous APIs for catalog and schema creation to unitycatalog-ai so that it's a one-stop shop for function creation. I'll add it to the tracker.